packages/spark-core/components/_alerts.scss

```diff
@@ -0,0 +1,64 @@
+// ==================================================================
+// Alerts
+// ==================================================================
+.sprk-c-Alert {
+  background-color: $gray-light;
+  border-radius: $corners;
+}
+
+.sprk-c-Alert,
+.sprk-c-Alert__icon {
+  display: flex;
+}
+
+.sprk-c-Alert__icon,
+.sprk-c-Alert__text {
+  padding: $space-inset-m;
+}
+
+.sprk-c-Alert__text {
+  align-self: center;
+  flex: 1;
+}
+
+.sprk-c-Alert__icon {
+  align-items: center;
+  justify-content: center;
+  width: 4.375rem;
```

Comment:
This width of 4.375rem is based on the current (master branch) width of the icon areas